src/inference/Inference.jl

```diff
@@ -123,19 +123,20 @@ end
 ######################
 # Default Transition #
 ######################
+# Default
+# Extended in contrib/inference/abstractmcmc.jl
+getstats(t) = nothing
 
 struct Transition{T, F<:AbstractFloat, S<:Union{NamedTuple, Nothing}}
     θ     :: T
     lp    :: F # TODO: merge `lp` with `stat`
     stat  :: S
 end
 
-Transition(θ, lp) = Transition(θ, lp, nothing)
```

Comment:
I think this is still needed and should not have been removed (maybe I suggested this incorrectly?) - it covers the cases where the first argument is not an `AbstractVarInfo`.